### DISCUSSION TITLE
Monitoring/observability setups that you have found useful

### DISCUSSION THREAD

**Discussion by georgiosd** [ravendb/ravendb#14245] [Q&A]
Hey everyone

Our systems have grown over time and we're finding it's time to introduce better observability measures, especially for our RavenDB cluster.

We're currently tracking exceptions with Sentry which we're finding to be amazing value for money, but we also need to monitor how the machines hosting the cluster are doing etc.

I know Raven supports SNMP and I looked around for solutions surrounding that but I've been overwhelmed by the amount of choice.

Any setups that you have found to be useful and would recommend? Can be paid, as long as they don't break the bank :)

Thanks!

Georgios

**Comment by ayende** [comment]
If you are running in the cloud, there are usually already systems there that you can utilize.
Azure Monitor can plug into SNMP, for example.

**Reply by georgiosd** [reply]
I should look into this further but their pricing policy will likely result in a larger bill than going open source. 

Scaleway is running a beta of a stack with Grafana and Loki that I'd like to test out but I'm curious as to what other people are using.
